subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -70,6 +70,32 @@ static sys_slist_t hbuf_pend;
 static s32_t hbuf_count;
 #endif
 
+static struct net_buf *process_prio_evt(struct node_rx_pdu *node_rx)
+{
+#if defined(CONFIG_BT_REMOTE_VERSION)
```

Comment:
This is new Kconfig? is the shippable failing on a dependent commit/PR?

Reply:
If so, then we have a circular dependency here.

Reply:
No, I will change this actually